packages/react-native/src/primitives/Label/types.ts

```diff
@@ -1,8 +1,45 @@
 import { TextProps, TextStyle } from 'react-native';
 
-export interface LabelProps extends TextProps {}
+export interface LabelProps extends TextProps {
+  /**
+   * @description
+   * The variation property will affect the color of the Label.
+   */
+  variation?: LabelVariation;
+}
 
 export type LabelPosition = 'start' | 'end' | 'top' | 'bottom';
+
+export type LabelVariation =
+  | 'primary'
+  | 'secondary'
+  | 'tertiary'
+  | 'error'
+  | 'warning'
+  | 'info'
+  | 'success';
+
+export type FontWeightType =
+  | 'normal'
+  | 'bold'
+  | '100'
+  | '200'
+  | '300'
+  | '400'
+  | '500'
+  | '600'
+  | '700'
+  | '800'
+  | '900'
+  | undefined;
```

Comment:
What happens if the `fontWeight` is `undefined`?

Reply:
The default value for font weight is `normal`: https://reactnative.dev/docs/text-style-props#fontweight